.storybook/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+module.exports = ({ config }) => {
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: "ts-loader",
+      },
+      {
+        loader: "react-docgen-typescript-loader",
+      },
+    ],
+  });
+
+  config.module.rules.push({
+    test: /\.(woff|woff2|eot|ttf)$/,
+    use: [
+      {
+        loader: "file-loader",
+        query: {
+          name: "[name].[ext]",
+        },
+      },
+    ],
+  });
+
+  config.module.rules.push({
+    test: /\.(svg|png|jpg)$/i,
+    use: [
+      {
+        loader: "url-loader",
+        query: {
+          name: "[name].[ext]",
+        },
+      },
+    ],
+  });
+
+  config.resolve.extensions.push(".ts", ".tsx", "woff2");
```

Comment:
I wonder if you'll need to include [module resolution](https://github.com/gnosis/safe-cmm-app-react/blob/f2cc4e1384f3d38031c830b8cfb5215c301c8bb4/webpack.config.js#L54) here too.

You can try adding an absolute import in Test component and see if it breaks.

Reply:
Spot on!

I've done it on the follow up PR https://github.com/gnosis/safe-cmm-app-react/pull/8/files#diff-6da49b24efd584154cfb4ec759745192R40